persistent/persistence.py

```diff
@@ -230,16 +252,16 @@ def _get_status(self):
     def __getattribute__(self, name):
         """ See IPersistent.
         """
-        if (not name.startswith('_Persistent__') and
```

Comment:
Why are we allowing activation for dunder-names?  Or is this a speed hack because you have eliminated all `self.__*` use in the module?